applications/nrf5340_audio/src/bluetooth/le_audio_bis_headset.c

```diff
@@ -39,16 +39,24 @@ struct audio_codec_info {
 struct active_stream {
 	struct bt_audio_stream *stream;
 	struct audio_codec_info *codec;
+	uint8_t brdcast_src_name_idx;
 };
 
+struct bt_name {
+	char *name;
+	size_t size;
+};
+
+static const char *const brdcast_src_names[] = { CONFIG_BT_DEVICE_NAME, CONFIG_BT_DEVICE_NAME_ALT };
+
 static struct bt_audio_broadcast_sink *broadcast_sink;
 
 static struct bt_audio_stream audio_streams[CONFIG_BT_AUDIO_BROADCAST_SNK_STREAM_COUNT];
 static struct bt_audio_stream *audio_streams_p[ARRAY_SIZE(audio_streams)];
 static struct audio_codec_info audio_codec_info[CONFIG_BT_AUDIO_BROADCAST_SNK_STREAM_COUNT];
 static uint32_t bis_index_bitfields[CONFIG_BT_AUDIO_BROADCAST_SNK_STREAM_COUNT];
 
-static struct active_stream active_audio_stream;
+static struct active_stream active_stream;
```

Comment:
I would try to avoid having the names being identical, very confusing potentially

Reply:
Also, there are a lot of structs here now with the same size, so we should probably do a rewrite to gather them in one struct rather than having multiple. Not in this PR though

Reply:
Changed the struct name to active_audio_stream.

Yes we should do a rewrite here